docs/fr/essentials/getting-started.md

```diff
@@ -64,6 +64,29 @@ const app = new Vue({
 // L'application est maintenant en marche !
 ```
 
+En injectant le routeur, nous y avons accès à travers `this.$router`. Nous avons également accès à la route courante derrière `this.$route` depuis n'importe quel composant :
+
+```js
+// Home.vue
+export default {
+  computed: {
+    username () {
+      // Nous verrons pourquoi `params` est court.
```

Comment:
Je n'ai pas compris cette phrase dans l'original. Ça disait « We will see what `params` is shortly ». Tu as une idée @posva ?

Reply:
ca veut dire _nous verrons ce que sont les params en un instant_

Reply:
je change ca, j'ai merge un peu trop vite 😅

Reply:
Merci @posva. Ça paraît tellement évident maintenant !